resources/rpm/project.spec.erb

```diff
@@ -68,7 +68,12 @@ Requires:  <%= requires %>
 # did not specify a dependency on these.
 # In the future, we will supress pre/post scripts completely if there's nothing
 # specified by the project or the components.
-<%- unless @platform.is_aix? -%>
+<%- if @platform.name =~ /fedora-29/ -%>
```

Comment:
This should be more generic. Future fedora releases will almost certainly be affected and, based on https://tickets.puppetlabs.com/browse/CPR-647, possibly previous fedora releases as well (though that ticket doesn't specify which OS they're actually using). @branan thinks possibly el 8 as well.

Reply:
Previous fedora releases don't seem to be affected (`coreutils` on fedora28 still provides /bin/mkdir etc). I'd like to see this work for fedora >= 29, but I wouldn't worry about pulling in el 8 things yet, that definitely feels like a different ticket/pr.